tests/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 # MDAL - Mesh Data Abstraction Library (MIT License)
 # Copyright (C) 2018 Peter Petrik (zilolv at gmail dot com)
 
+IF (HDF5_FOUND)
```

Comment:
this is not required, the compiler definitions are already generated by https://github.com/lutraconsulting/MDAL/blob/8ea01b603b21d929c61c4b5d0f85830099011b0f/CMakeLists.txt#L103 in file mdal_config.hpp 

It is enough in the test to include this file and use macro HAVE_HDF5

Reply:
Using HDF5_FOUND  was my first idea but it did not work - it alwas thought the HDF5 was included. Thus the slight cludge.

But - I had not spotted the use of ifdef HAVE_HDF5. I will try that and see if I can get it to work.

Reply:
one is in the cmake , other in c++ files

Reply:
Ok - that is done and looks like it is working